products/views.py

```diff
@@ -1,3 +1,27 @@
-from django.shortcuts import render
+from django.http  import JsonResponse
+from django.views import View
+
+from products.models import Product
+
+class ProductDetailView(View):
+    def get(self, request, product_id):
+        try:
+            product = Product.objects.prefetch_related('productimage_set').get(id=product_id)
+            options = product.options.filter(product__id=product_id)
```

Comment:
```suggestion
            options = product.options.all()
```
이미 Many to Many field가 걸려 있고, 특정 product에서 option을 찾는 것이기 때문에 위와 같이 수정해도 됩니다. django shell에서 한 번 확인해 보고 결과에 이상이 없으면 수정해 주세요.

Reply:
아하 넵 ㅠㅠ 저도 불필요하다 생각했는데 all을 깜빡했네요 수정하겠습니다!